### ISSUE TITLE
getting wrong number of lines when ending within the same minute

### ISSUE THREAD

**Issue by stefco** [stefco/geco_data#19]
maybe i'm not handling start/stop on same frame file properly after all.

```
$ geco_dump -s 'Nov 1 2015' -e 'Nov 1 00:00:10 2015' -c H1:CAL-PCALX_IRIGB_OUT_DQ -T > /dev/null
+ slow_channels=("H1:SYS-TIMING_C_MA_A_PORT_2_SLAVE_CFC_TIMEDIFF_1" "H1:SYS-TIMING_X_FO_A_PORT_9_SLAVE_CFC_TIMEDIFF_1" "H1:SYS-TIMING_Y_FO_A_PORT_9_SLAVE_CFC_TIMEDIFF_1" "H1:SYS-TIMING_C_MA_A_PORT_2_SLAVE_CFC_TIMEDIFF_2" "H1:SYS-TIMING_X_FO_A_PORT_9_SLAVE_CFC_TIMEDIFF_3" "H1:SYS-TIMING_Y_FO_A_PORT_9_SLAVE_CFC_TIMEDIFF_3" "L1:SYS-TIMING_C_MA_A_PORT_2_SLAVE_CFC_TIMEDIFF_1" "L1:SYS-TIMING_X_FO_A_PORT_9_SLAVE_CFC_TIMEDIFF_1" "L1:SYS-TIMING_Y_FO_A_PORT_9_SLAVE_CFC_TIMEDIFF_1" "L1:SYS-TIMING_C_MA_A_PORT_2_SLAVE_CFC_TIMEDIFF_2" "L1:SYS-TIMING_X_FO_A_PORT_9_SLAVE_CFC_TIMEDIFF_3" "L1:SYS-TIMING_Y_FO_A_PORT_9_SLAVE_CFC_TIMEDIFF_3")
+ outdir_pre=
+ start_time=
+ end_time=
+ check_dump_progress=false
+ read_from_dumped_files=false
+ print_timeseries_string=false
+ unset channel_names
+ getopts :p:s:e:c:l:PTRh opt
+ case ${opt} in
+ start_time='Nov 1 2015'
+ getopts :p:s:e:c:l:PTRh opt
+ case ${opt} in
+ end_time='Nov 1 00:00:10 2015'
+ getopts :p:s:e:c:l:PTRh opt
+ case ${opt} in
+ channel_names+=("${OPTARG}")
+ getopts :p:s:e:c:l:PTRh opt
+ case ${opt} in
+ print_timeseries_string=true
+ getopts :p:s:e:c:l:PTRh opt
+ shift 7
+ ((  0  ))
+ '[' 0 == 0 ']'
+ '[' 'Nov 1 2015' == '' ']'
+ '[' 'Nov 1 00:00:10 2015' == '' ']'
+ '[' 1 == 0 ']'
+ true
+ false
+ true
+ '[' 1 == 1 ']'
++ lalapps_tconvert Nov 1 2015
+ gps_start_time=1130371217
++ lalapps_tconvert Nov 1 00:00:10 2015
+ gps_end_time=1130371227
+ false
+ for channel_name in '${channel_names[*]}'
++ get_outdir '' H1:CAL-PCALX_IRIGB_OUT_DQ
++ outdir_pre=
++ channel_name=H1:CAL-PCALX_IRIGB_OUT_DQ
++ '[' '' == '' ']'
+++ printf H1:CAL-PCALX_IRIGB_OUT_DQ
+++ tr : _
++ printf /home/stefan.countryman/H1_CAL-PCALX_IRIGB_OUT_DQ
+ outdir=/home/stefan.countryman/H1_CAL-PCALX_IRIGB_OUT_DQ
+ true
+ dump 1130371217 1130371227 /home/stefan.countryman/H1_CAL-PCALX_IRIGB_OUT_DQ H1:CAL-PCALX_IRIGB_OUT_DQ true false
+ gps_start_time=1130371217
+ gps_end_time=1130371227
+ outdir=/home/stefan.countryman/H1_CAL-PCALX_IRIGB_OUT_DQ
+ channel_name=H1:CAL-PCALX_IRIGB_OUT_DQ
+ print_timeseries_string=true
+ read_from_dumped_files=false
++ dc -e '1130371217 64/64*p'
+ dump_start_time=1130371200
++ dc -e '1130371227 64/64*64+p'
+ dump_end_time=1130371264
+ current_time=1130371200
+ '[' -e /home/stefan.countryman/H1_CAL-PCALX_IRIGB_OUT_DQ ']'
++ echo H1:CAL-PCALX_IRIGB_OUT_DQ
++ head -c 1
+ observatory_letter=H
+ false
+ true
++ get_frame_file_path H 1130371200
++ observatory_letter=H
++ current_time=1130371200
++ gw_data_find -o H -t H1_R -s 1130371200 -e 1130371200 -u file
++ sed 's/[^:^\/]*:\/\/[^\/]*//'
+ inpath=/data/node45/frames/O1/raw/H1/H-H1_R-113037/H-H1_R-1130371200-64.gwf
++ mktemp
+ buffer=/usr1/stefan.countryman/tmp.yJZA0nZoiG
+ framecpp_dump_channel --channel H1:CAL-PCALX_IRIGB_OUT_DQ /data/node45/frames/O1/raw/H1/H-H1_R-113037/H-H1_R-1130371200-64.gwf
++ sed '5s/[NDat: ]//g;5q;d' /usr1/stefan.countryman/tmp.yJZA0nZoiG
+ bitrate=16384
+ '[' 1130371200 -lt 1130371264 ']'
+ outpath=/home/stefan.countryman/H1_CAL-PCALX_IRIGB_OUT_DQ/1130371200.dat
+ false
+ true
++ get_frame_file_path H 1130371200
++ observatory_letter=H
++ current_time=1130371200
++ gw_data_find -o H -t H1_R -s 1130371200 -e 1130371200 -u file
++ sed 's/[^:^\/]*:\/\/[^\/]*//'
+ inpath=/data/node45/frames/O1/raw/H1/H-H1_R-113037/H-H1_R-1130371200-64.gwf
+ '[' /data/node45/frames/O1/raw/H1/H-H1_R-113037/H-H1_R-1130371200-64.gwf == '' ']'
+ '[' 1130371200 -eq 1130371200 ']'
+ cat /usr1/stefan.countryman/tmp.yJZA0nZoiG
+ check_and_dump
+ sed -n '
        2{/H1:CAL-PCALX_IRIGB_OUT_DQ/!q1}
        7s/[Dat: ]//g
        s/,/\
/gp
+ cut_lines 16384 1130371200 1130371217 1130371227
    '
+ bitrate=16384
+ current_time=1130371200
+ gps_start_time=1130371217
+ gps_end_time=1130371227
++ dc -e '1130371217 64/64*p'
+ dump_start_time=1130371200
++ dc -e '1130371227 64/64*64+p'
+ dump_end_time=1130371264
+ del_at_start=
+ del_at_end=
+ '[' 1130371200 -eq 1130371200 ']'
+ del_at_start='1,278528d;'
+ sed '1,278528d;'
+ let current_time+=64
+ '[' 1130371264 -lt 1130371264 ']'
```

**Comment by stefco** [comment]
`elif` should be another `if` in `cut_lines`

**Comment by stefco** [comment]
this fixed it

**Comment by stefco** [comment]
Fixed in fd4e5f7
